src/test/java/com/feelmycode/parabole/controller/CouponControllerTest.java

```diff
@@ -0,0 +1,363 @@
+                        fieldWithPath("data.last").type(JsonFieldType.BOOLEAN)
+                            .description("마지막 페이지 여부"),
+                        fieldWithPath("data.size").type(JsonFieldType.NUMBER).description("페이지당 항목 수"),
+                        fieldWithPath("data.number").type(JsonFieldType.NUMBER).description("현재 페이지"),
+                        fieldWithPath("data.sort").type(JsonFieldType.OBJECT).description("정렬 정보"),
+                        fieldWithPath("data.sort.empty").type(JsonFieldType.BOOLEAN)
+                            .description("정렬 정보 여부"),
+                        fieldWithPath("data.empty").type(JsonFieldType.BOOLEAN).description("빈 여부")
+                    )
+                )
+            )
+            .when()
+            .port(port)
+            .get("/api/v1/coupon/list");
+
+        // Then
+        Assertions.assertEquals(HttpStatus.OK.value(), resp.statusCode());
+    }
+
+    @Test
+    @DisplayName("셀러 새로운 쿠폰 등록")
+    public void addCoupon() {
+
+        Long userId = 37L;
+        String userToken = this.tokenProvider.create(
+            User.builder().seller(new Seller()).id(userId).email("s1-u2@naver.com").build());     // 셀러 계정
+
+        LocalDateTime validAt = LocalDateTime.parse("2022-09-16T00:00:00", DateTimeFormatter.ISO_LOCAL_DATE_TIME);
+        LocalDateTime expiresAt = LocalDateTime.parse("2022-11-30T00:00:00", DateTimeFormatter.ISO_LOCAL_DATE_TIME);
+
+        CouponCreateRequestDto dto = new CouponCreateRequestDto("크리스마스 기념 쿠폰", userId, 2, 2000,
+            validAt, expiresAt, "행복한 크리스마스 되세요", 5);
+
+        // When
+        Response resp = given(this.spec)
+            .accept(ContentType.JSON)
+            .header("authorization", "Bearer " + userToken)
+            .body(dto)
+            .contentType(ContentType.JSON)
+            .filter(document("coupon-add-seller",
+                    preprocessRequest(modifyUris()
+                            .scheme("https")
+                            .host("parabole.com"),
+                        prettyPrint()),
+                    preprocessResponse(prettyPrint()),
+                    responseFields(
+                        fieldWithPath("success").type(JsonFieldType.BOOLEAN).description("성공여부"),
+                        fieldWithPath("message").type(JsonFieldType.STRING).description("메세지"),
+                        fieldWithPath("data").type(JsonFieldType.OBJECT).description("등록한 쿠폰 정보"),
+                        fieldWithPath("data.couponId").type(JsonFieldType.NUMBER)
+                            .description("쿠폰 ID"),
+                        fieldWithPath("data.couponName").type(JsonFieldType.STRING)
+                            .description("쿠폰 이름"),
+                        fieldWithPath("data.sellerName").type(JsonFieldType.STRING)
+                            .description("판매자 이름"),
+                        fieldWithPath("data.type").type(JsonFieldType.STRING)
+                            .description("쿠폰 유형"),
```

Comment:
쿠폰 유형에 어떤것이 있는지 설명에 작성해주세요

아래는 예시이고 알맞게 수정해주세요

```suggestion
                        fieldWithPath("data.type").type(JsonFieldType.STRING)
                            .description("쿠폰 유형 (정률쿠폰: RATE / 정액쿠폰: AMOUNT)"),
```